### PR TITLE
chore(medusa,admin-ui): Add reservations FF

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

**Review by fPolic — APPROVED** [review]
LGTM

File: packages/medusa/src/loaders/feature-flags/reservations.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { FlagSettings } from "../../types/feature-flags"
+
+const ReservationsFeatureFlag: FlagSettings = {
+  key: "reservations",
+  default_val: false,
+  env_key: "MEDUSA_FF_RESERVATIONS",
+  description:
+    "Enable Medusa to collect data on usage, errors and performance for the purpose of improving the product",
```
Comment (srindom) [inline packages/medusa/src/loaders/feature-flags/reservations.ts]: todo: update for reservations

**Comment by olivermrbl** [comment]
good to merge @srindom ?

**Review by srindom — APPROVED** [review]
LGTM

**Pull request merged**